generate-analytical-dataset.py

```diff
@@ -96,6 +99,16 @@ def getPrinted(values):
     for x in values:
         print(x)
 
+def getTables(db_name):
+    table_list = []
+    client = boto3.client("glue")
+    tables_metadata_dict = client.get_tables(DatabaseName = db_name)
+    tables_dict = tables_metadata_dict["TableList"]
+    for table_dict in tables_dict:
+       table_list.append(table_dict["Name"])
+    return table_list
+
+
```

Comment:
this method may be unnecessary if you can iterate over the dictionary returned by client.get_tables(). Could save time not doing this additional conversion to a list